packages/reports/tests/acceptance/schedule-modal-test.js

```diff
@@ -244,6 +262,9 @@ module('Acceptance | Navi Report Schedule Modal', function(hooks) {
     await click('.schedule-modal__dropdown--frequency .ember-power-select-trigger');
     await click($('.ember-power-select-option:contains(Day)')[0]);
 
+    // Set notify when empty to be true
```

Comment:
remove comment.

Reply:
Are we going to update these comments? The component is called `must have data` instead of `notify when empty` now so the comments don't really make sense